Common/DTO/BrokerRequests/InternalEditUserInfoRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿
+namespace DTO.BrokerRequests
+{
+    public class InternalEditUserInfoRequest
+    {
+        public User User { get; set; }
+        public UserPasswordHashChange UserPasswords { get; set;}
```

Comment:
Maybe change this name to PasswordHashChange or another shorter one?

Reply:
Yeah, I was thinking abou that , but not sure. At least, it is one of user models . Who else how are thinking?

Reply:
`OldNewPasswordHash`? `PasswordChangeRequest`?
Idk about those, but I also don't really like current one :)

Reply:
Suggestions?

Reply:
> Suggestions?

There're 2 already

Reply:
fixed